src/FakeItEasy/Configuration/IRepeatConfiguration.cs

```diff
@@ -9,7 +9,7 @@ public interface IRepeatConfiguration<out TInterface> : IHideObjectMembers
         /// <summary>
         /// Specifies the number of times for the configured event.
         /// </summary>
-        /// <param name="numberOfTimesToRepeat">The number of times to repeat.</param>

```

Comment:
I missed that when I reviewed... You changed the parameter name in the doc comments, but not in the method signature itself ;)
I'd like to change the name, but it's a public API, so technically it would be a breaking change... Not a big one, though; I doubt anyone writes `.NumberOfTimes(numberOfTimesToRepeat: 3)`.
Anyway, I'd like to change the method description as well. It's weird that it mentions an "event". Something like "the number of times the configured behavior should be applied" would be better, I think.